app/api/routers/breeze_buddy/test/handlers.py

```
@@ -0,0 +1,291 @@
+        current_user.username,
+        body.mode,
+        template_id,
+        len(body.scenarios),
+    )
+    template = await _fetch_template_or_404(template_id)
+    run_id = str(uuid.uuid4())
+
+    if body.mode == TestMode.STRUCTURAL:
+        try:
+            results = await run_scenarios_structural(template, body.scenarios)
+        except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)